development/tools/jobs/acceptance_tests_test.go

```diff
@@ -0,0 +1,62 @@
+package jobs_test
+
+import (
+	"testing"
+
+	"github.com/kyma-project/test-infra/development/tools/jobs/tester"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"k8s.io/test-infra/prow/kube"
+)
+
+func TestAcceptanceJobsPresubmit(t *testing.T) {
+	// WHEN
+	jobConfig, err := tester.ReadJobConfig("./../../../prow/jobs/kyma/tests/acceptance/acceptance.yaml")
+	// THEN
+	require.NoError(t, err)
+
+	assert.Len(t, jobConfig.Presubmits, 1)
+	kymaPresubmits, ex := jobConfig.Presubmits["kyma-project/kyma"]
+	assert.True(t, ex)
+	assert.Len(t, kymaPresubmits, 1)
+
+	actualPresubmit := kymaPresubmits[0]
+	expName := "prow/kyma/tests/acceptance"
+	assert.Equal(t, expName, actualPresubmit.Name)
+	assert.Equal(t, []string{"master"}, actualPresubmit.Branches)
+	assert.Equal(t, 10, actualPresubmit.MaxConcurrency)
+	assert.True(t, actualPresubmit.SkipReport)
+	assert.True(t, actualPresubmit.Decorate)
+	assert.Equal(t, "github.com/kyma-project/kyma", actualPresubmit.PathAlias)
```

Comment:
please add an assertion that checks run_if_changed field

Reply:
it's already there `assert.Equal(t, "^tests/acceptance/", actualPresubmit.RunIfChanged)`#L33 :)